NAMESPACE

```diff
@@ -1,47 +1,46 @@
 # Generated by roxygen2: do not edit by hand
 
 export("%>%")
-export(AboveNationalWQXUpperThreshold)
-export(AggregatedContinuousData)
-export(BelowNationalWQXLowerThreshold)
-export(GetActivityTypeRef)
-export(GetDetCondRef)
-export(GetDetLimitRef)
-export(GetMeasureUnitRef)
-export(GetWQXCharValRef)
-export(HarmonizationRefTable)
-export(HarmonizeData)
-export(InvalidCoordinates)
-export(InvalidFraction)
-export(InvalidMethod)
-export(InvalidResultUnit)
-export(InvalidSpeciation)
-export(JoinWQPProfiles)
-export(PotentialDuplicateRowID)
-export(QAPPDocAvailable)
-export(QAPPapproved)
-export(QualityControlActivity)
-export(TADABigdataRetrieval)
-export(TADAOverviewMap)
-export(TADAReadWQPWebServices)
+export(TADA_AboveNationalWQXUpperThreshold)
```

Comment:
Should we shorten this and the lower threshold function name? e.g. TADA_AboveUpperThreshold

Reply:
I'm into it. Sounds good. And same with BelowLower.